web/modules/custom/helloworld/helloworld.module

```diff
@@ -0,0 +1,29 @@
+<?php 
+
+use Drupal\Core\StringTranslation\StringTranslationTrait;
+
+/**
+ * @file
+ * Implements hooks.
+ */
+
+/**
+ * Implements hook form alter.
+ *
+ * @param array $form
+ * @param \Drupal\Core\Form\FormStateInterface $form_state
+ *   The FormStateInterface object.
+ * @param string $form_id
+ * @return void
+ */
+function helloworld_form_alter(&$form, \Drupal\Core\Form\FormStateInterface $form_state, $form_id) {
```

Comment:
We don't mention namespace in the function arguments.
Please check how it is done in Core.